presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,31 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table
```

Comment:
(for some reason) we allow `SHOW STATS ON table` as well.
Maybe we should drop one and retain only FOR?

@kokosing @martint